examples/completions/generate_completions.py

```diff
@@ -0,0 +1,134 @@
+import asyncio
```

Comment:
Great work! Maybe some doc strings in the beginning of the script, like how to run the script. How to resume from intermediate results. How to submit to slurm, etc